I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 13, in script
    image background_video_interogation = Moive(play = "video/movie_interogation.webm", mask = None)
  File "game/script.rpy", line 13, in script
    image background_video_interogation = Moive(play = "video/movie_interogation.webm", mask = None)
  File "game/script.rpy", line 13, in <module>
    image background_video_interogation = Moive(play = "video/movie_interogation.webm", mask = None)
NameError: name 'Moive' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "/Applications/renpy-7.4.10-sdk/renpy/bootstrap.py", line 331, in bootstrap
    renpy.main.main()
  File "/Applications/renpy-7.4.10-sdk/renpy/main.py", line 550, in main
    renpy.game.context().run(node)
  File "game/script.rpy", line 13, in script
    image background_video_interogation = Moive(play = "video/movie_interogation.webm", mask = None)
  File "game/script.rpy", line 13, in script
    image background_video_interogation = Moive(play = "video/movie_interogation.webm", mask = None)
  File "/Applications/renpy-7.4.10-sdk/renpy/ast.py", line 1027, in execute
    img = renpy.python.py_eval_bytecode(self.code.bytecode)
  File "/Applications/renpy-7.4.10-sdk/renpy/python.py", line 2269, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 13, in <module>
    image background_video_interogation = Moive(play = "video/movie_interogation.webm", mask = None)
NameError: name 'Moive' is not defined

Darwin-20.4.0-x86_64-i386-64bit
Ren'Py 7.4.10.2178
eastern_medicine_project 1.0
Wed Nov 17 15:58:25 2021
